<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAccountsMgrWin</class>
 <widget class="QDialog" name="CAccountsMgrWin">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>379</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>账户管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="layout1">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListView" name="m_iAccountsNameView">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="m_iAddBtn">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_iDelBtn">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="m_iDetailWidget">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <widget class="QWidget" name="tab1">
        <attribute name="title">
         <string>常规</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>账户</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <property name="spacing">
             <number>5</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>类型：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="m_iAccountTypeEdit">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>币种：</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="m_iCurrencyEdit">
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Start Balance：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" rowspan="2">
             <widget class="QDoubleSpinBox" name="m_iStartBalanceEdit">
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="maximum">
               <double>100000000000000000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="0" rowspan="2">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>备注：</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QTextEdit" name="m_iRemarkEdit">
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QCheckBox" name="m_iAccountClosedEdit">
              <property name="text">
               <string>此账户已关闭</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>机构</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>名称：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="m_iInstutionNameEdit"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>账户号码：</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="m_iInstutionNumEdit"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab2">
        <attribute name="title">
         <string>详细</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Balance Limits</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Overdraft at:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m_iOverdraftBalanceEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Miscellaneous</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Default Template:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="m_iTemplateEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Miscellaneous</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCheckBox" name="m_iExcludeSumEdit">
              <property name="text">
               <string>从账户汇总中排除</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="m_iExcludeBudgetEdit">
              <property name="text">
               <string>从预算中删除</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="m_iExcludeReportsEdit">
              <property name="text">
               <string>从报告中排除</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_iApplyBtn">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_iCloseBtn">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_iAccountsNameView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>CAccountsMgrWin</receiver>
   <slot>__On_AccountName_Select(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_iAccountClosedEdit</sender>
   <signal>stateChanged(int)</signal>
   <receiver>CAccountsMgrWin</receiver>
   <slot>__On_AccountClose_State_Changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_iApplyBtn</sender>
   <signal>clicked()</signal>
   <receiver>CAccountsMgrWin</receiver>
   <slot>__On_ApplyBtn_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_iCloseBtn</sender>
   <signal>clicked()</signal>
   <receiver>CAccountsMgrWin</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_iAddBtn</sender>
   <signal>clicked()</signal>
   <receiver>CAccountsMgrWin</receiver>
   <slot>__On_AddBtn_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_iDelBtn</sender>
   <signal>clicked()</signal>
   <receiver>CAccountsMgrWin</receiver>
   <slot>__On_DelBtn_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>__On_AccountName_Select(QModelIndex)</slot>
  <slot>__On_AccountClose_State_Changed(int)</slot>
  <slot>__On_ApplyBtn_Clicked()</slot>
  <slot>__On_AddBtn_Clicked()</slot>
  <slot>__On_DelBtn_Clicked()</slot>
 </slots>
</ui>
